apolloschurchapp/src/tabs/index.js

```diff
@@ -111,20 +104,46 @@ const HomeTab = createFeatureFeedTab({
 });
 
 const ReadTab = createFeatureFeedTab({
+  screenOptions: {
+    headerLeft: HeaderLeft,
+  },
   tabName: 'Read',
   feedName: 'READ',
 });
 
 const WatchTab = createFeatureFeedTab({
+  screenOptions: {
+    headerLeft: HeaderLeft,
+  },
   tabName: 'Watch',
   feedName: 'WATCH',
 });
 
 const PrayTab = createFeatureFeedTab({
+  screenOptions: {
+    headerLeft: HeaderLeft,
+  },
   tabName: 'Pray',
   feedName: 'PRAY',
 });
 
+const CustomConnectScreen = () => (
+  <ConnectScreenConnected ActionTable={ActionTable} ActionBar={ActionBar} />
+);
+
+const ConnectTabStack = createNativeStackNavigator();
+const ConnectTabStackNavigator = () => (
+  <ConnectTabStack.Navigator
+    screenOptions={{
+      headerHideShadow: true,
```

Comment:
```suggestion
```

Reply:
(don't think we need to hide the shadow just on this tab? We don't on any other tab)